mindsdb_native/libs/backends/lightwood.py

```diff
@@ -125,7 +125,7 @@ def _create_lightwood_config(self):
                 self.transaction.log.error(f'The lightwood model backend is unable to handle data of type {data_type} and subtype {data_subtype} !')
                 raise Exception('Failed to build data definition for Lightwood model backend')
 
-            if col_name in [x[0] for x in self.transaction.lmd['model_order_by']]:
+            if self.transaction.lmd['tss']['is_timeseries'] and col_name in self.transaction.lmd['tss']['order_by']:
```

Comment:
1. here it's assumed that ['tss']['order_by'] is a list
(see next comment)

Reply:
It isn't assumed it's a list, it's assumed it's a list if `self.transaction.lmd['tss']['is_timeseries']  == True`